src/objects/Trigger_Handler__c.object

```diff
@@ -89,6 +89,16 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Exclude_For_These_Usernames__c</fullName>
```

Comment:
@smarovich I was also going to ask for a string review.  Michael has used the developer name Exclude_For_These_Usernames__c, but the label is Do Not Run for these Users.  Typically, we try to keep the dev name and label the same.  Also, this name seems a bit longer and more phrase like than we have done in the past.  I'm not against it, but it does seem a bit different than most fields in NPSP.